Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM lukemathwalker/cargo-chef:latest-rust-1 AS chef
+FROM docker.io/lukemathwalker/cargo-chef:0.1.67-rust-1.78.0-bookworm AS chef
```

Comment:
Is this possible to do with `latest` instead of pinning to a rust version? I'd prefer it run latest so if it fails / breaks, we know immediately instead of relying on bumping.